app/src/navigation/screens/Home.tsx

```diff
@@ -39,13 +39,12 @@ const Home: React.FC = () => {
   const addressValue = usersClaims.find(
     (claim) => claim.type === ClaimTypeConstants.AddressCredential
   )?.value;
-
   const { verifyClaims } = useVerifyClaims();
   const verifyUserOnProxy = async () => {
     if (splitName && dob && address && email) {
```

Comment:
Just personal preference here - but using a guard clause, `if (!splitName || !dob || !address || !email) return;`, would look cleaner and prevent a line of indentation - makes everything a bit nicer to read.